posts/2022-09-22-real-time-everything.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Real-time “everything” with operational and analytics data'

```

Comment:
Vendia site/blog magic will use this to create an HTML header  that will serve as the title